commcare-cloud/commcare_cloud/commcare_cloud.py

```diff
@@ -46,6 +45,13 @@ def arg_branch(parser):
     ))
 
 
+def arg_stdout_callback(parser, default='default'):
+    parser.add_argument(
```

Comment:
Worth including something like

```
default = os.environ.get('ANSIBLE_STDOUT_CALLBACK') or default
```
before this line in case people want to specify `ANSIBLE_STDOUT_CALLBACK=minimal` (possibly in their bash profile) to change the global default? (And generally to degrade gracefully to current behavior.) You may be the only person who uses this so far, so if that doesn't sound useful to you, no need to block.

Reply:
6e5b059